jekyll/_cci2/circleci-config-sdk.adoc

```diff
@@ -0,0 +1,61 @@
+---
+version:
+- Cloud
+---
+= CircleCI Config SDK
+:page-layout: classic-docs
+:page-liquid:
+:page-description: The CircleCI Config SDK is a JavaScript package library for generating CircleCI config YAML.
+:icons: font
+:toc: macro
+:toc-title:
+
+[#overview]
+== Overview
+
+The link:https://circleci-public.github.io/circleci-config-sdk-ts[CircleCI Config SDK] is a JavaScript package library (also compatible with TypeScript) for generating a CircleCI YAML configuration file. Use the SDK to:
+
+* Replace writing YAML conguration files
+* Generate a static YAML configuration file
+* Modularize and manage your configuration as JavaScript packages
+* Enhance CircleCI's <<using-dynamic-configuration#,dynamic configuration>>
+* Build integrations with CLI tools or browser-based experiences
+
+For example, generating a static configuration file can be useful for CLI tools where you want to create a CircleCI configuration, or browser-based tooling, such as a link:https://github.com/CircleCI-Public/visual-config-editor/[visual config editor].
+
+Components created with the CircleCI Config SDK can be packaged and distributed as Node packages.
+
+[#install-the-sdk]
+== Install the SDK
+
+Install the CircleCI Config SDK with NPM or Yarn, and include it in your project via Node.js or the browser.
+
+NPM installation:
```

Comment:
I don't think we have to implement it here (and we should probably have a larger discussion on it as well) but is this the kind of snippet where tabs would work well?

Reply:
Yeah, that's a good idea!